app/graphql/mutations/team_invite/create_invite.rb

```diff
@@ -25,9 +25,23 @@ def resolve(emails:, team_id:)
         team_invites << team_invite
       end
 
+      error_messages = []
+      
       ::TeamInvite.transaction do
-        team_invites.each(&:save)
-        { team_invites: team_invites, errors: [] }
+        team_invites.each do |team_invite|
+          next if team_invite.save
+           
+          if team_invite.errors.any?
+            error_messages << team_invite.errors.full_messages.join(', ')
+          end
+        end
+
+        if error_messages.empty?
+          { team_invites: team_invites, errors: [] }
```

Comment:
It looks like the manually added 'errors' field inside the response is redundant right?
Yesterday we saw that the field was empty in the chrome request inspector even though the API was  returning an error. This might be confusing while debugging.

Reply:
> It looks like the manually added 'errors' field inside the response is redundant right?

Yes that is correct. This is the way it is described in the docs of the GraphQL Ruby implementation:
https://graphql-ruby.org/mutations/mutation_classes.html

However, this way the Apollo client doesn't recognise the errors. In the case below `error` is only found when the backend is raising a `GraphQL::ExecutionError`. This means either change the implementation in the frontend or in the backend. But that might be something you can look into. 
```
{(createInvite, { error, loading }) => {
  ...
}
```